doc/whats_new/v0.20.rst

```diff
@@ -100,3 +100,19 @@ Linear, kernelized and related models
 - Deprecate ``random_state`` parameter in :class:`svm.OneClassSVM` as the
   underlying implementation is not random.
   :issue:`9497` by :user:`Albert Thomas <albertcthomas>`.
+
+Outlier Detection models
+
+- Add a ``score_samples`` method in :class:`svm.OneClassSVM`,
+  :class:`ensemble.IsolationForest`, :class:`neighbors.LocalOutlierFactor`,
+  :class:`covariance.EllipticEnvelope`. It allows to  access raw score
+  functions from original papers. A new ``offset_`` parameter allows to link
+  ``score_samples`` and ``decision_function`` methods.
+  :class:`ensemble.IsolationForest` and :class:`neighbors.LocalOutlierFactor`
+  ``decision_function`` methods now take a ``contamination`` parameter, used
+  to define a drift such that outliers (resp. inliers) have negative (resp.
+  positive) ``decision_function`` values.
```

Comment:
we should also mention that `contamination` is now set to `"auto"` for LOF and IsolationForest as this can change the output of `fit_predict` and `predict`.